src/particle/pdgid/functions.py

```diff
@@ -99,12 +101,37 @@ def is_quark(pdgid):
     return 1 <= abspid(pdgid) <= 8
 
 
+def is_sm_quark(pdgid):
+    # type: (PDGID_TYPE) -> bool
+    """
+    Does this PDG ID correspond to a Standard Model quark?
+
+    Codes 1-8 are reserved for Standard Model or fourth-generation quarks,
+    (excluding excited, i.e. composite, quarks)
+    but only the codes 1-6 actually correspond to SM quarks.
```

Comment:
```suggestion
	Fourth-generation leptons are not included.
    Codes 1-8 are reserved for Standard Model or fourth-generation quarks,
    (excluding excited, i.e. composite, quarks)
    but only the codes 1-6 actually correspond to SM quarks.
```

Maybe make it clearer that they are not "sm"?

Reply:
@jonas-eschle, the suggestion indicates the very same text or it is my eyes :S?

Reply:
There is a "Fourth-generation leptons are not included." in the beginning. I thought it may could be a bit clearer that this function does not return the fourth generation. The docs says about which codes are for what, but not which codes the function actually returns (kinda, implicitly, by explicitly saying "SM or fourth-generation".

Reply:
When reading it, I just didn't directly realise what the function now actually returns, although technically it is written. Just a bit more explicitly could be good, but maybe it's just me.

So also like this it's fine. My suggestion would be to just make it a bit clearer what the "sm" means here

Reply:
OK, will reread and commit a little update. Thanks for the comments - docs are really important :-).

Reply:
Done with my docstring update. I will now proceed and merge since Henry has a clean-up MR in the pipeline. Feel 100% free to comment further or suggestion other updates :-). Thanks again.